runtime/decimal.c

```diff
@@ -275,8 +275,7 @@ static bool decimalListContains(const DecimalConstPtr *start, const DecimalConst
     initContext(&cx);
     while (start < end) {
         const DecimalConstPtr *mid = start + (end - start)/2;
-        decQuad midVal;
-        decQuadFromString(&midVal, *mid, &cx);
+        decQuad midVal = (**mid);
```

Comment:
The function's comment needs changing.

Don't need the parentheses.

Reply:
fixed